libraries/src/HTML/Helpers/Select.php

```diff
@@ -631,7 +631,15 @@ public static function radiolist(
         $idtag = false,
         $translate = false
     ) {
+        $class = '';
+
         if (\is_array($attribs)) {
+
+            if (array_key_exists('class', $attribs)) {
```

Comment:
```suggestion
            if (\array_key_exists('class', $attribs)) {
```
Fix PHPCS (php-cs-fixer) in drone - native_function_invocation